etc/profile-m-z/strawberry.profile

```diff
@@ -0,0 +1,33 @@
+# Firejail profile for strawberry
+# Description: A music player and music collection organizer
+# This file is overwritten after every install/update
+# Persistent local customizations
+include strawberry.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.cache/strawberry
+noblacklist ${HOME}/.config/strawberry
```

Comment:
Add to `disbale-programs.inc`.

Reply:
I've committed again, please check and mention if anything else needs to be changed, thanks.